src/Uno.UI/UI/Xaml/Input/DoubleTappedRoutedEventArgs.cs

```diff
@@ -10,33 +10,36 @@ namespace Windows.UI.Xaml.Input
 {
 	public sealed partial class DoubleTappedRoutedEventArgs : RoutedEventArgs, ICancellableRoutedEventArgs
 	{
+		private readonly UIElement _originalSource;
 		private readonly Point _position;
 
 		public DoubleTappedRoutedEventArgs() { }
 
-		internal DoubleTappedRoutedEventArgs(object originalSource, TappedEventArgs args)
+		internal DoubleTappedRoutedEventArgs(UIElement originalSource, TappedEventArgs args)
 			: base(originalSource)
 		{
+			_originalSource = originalSource;
 			PointerDeviceType = args.PointerDeviceType;
 			_position = args.Position;
 		}
 
-		public Point GetPosition()
-			=> _position;
-
 		public bool Handled { get; set; }
 
-		public PointerDeviceType PointerDeviceType { get; internal set; }
+		public PointerDeviceType PointerDeviceType { get; }
 
 		public Point GetPosition(UIElement relativeTo)
 		{
-			if (relativeTo == null)
+			if (_originalSource == null)
+			{
+				return default; // Required for the default public ctor ...
+			}
+			else if(relativeTo == _originalSource)
 			{
 				return _position;
 			}
 			else
 			{
-				return relativeTo.GetPosition(_position, relativeTo);
+				return _originalSource.GetPosition(_position, relativeTo);
```

Comment:
So `relativeTo` is not used anymore?

Reply:
It is ... at the end of the line ... that was the issue!